### PR TITLE
Wiki pages integrated in gh-pages

### PR DESCRIPTION
**Pull request by MrTopom**

### CODE REVIEW

File: _data/sidebars/home_sidebar.yml [hunk 1]
```diff
@@ -3,20 +3,20 @@
...
+#      output: web
+#    - title: Theme instructions
+#      url: /mydoc_introduction.html
+#      output: web
+#    - title: Product 1
+#      url: /p1_landing_page.html
+#      output: web
+#    - title: Product 2
+#      url: /p2_landing_page.html
+#      output: web
```
Comment (simlu) [inline _data/sidebars/home_sidebar.yml]: Why is this commented and not deleted? Do we still have plans for it?

File: _data/topnav.yml [hunk 2]
```diff
@@ -5,28 +5,15 @@ topnav:
...
-          url: /mydoc_introduction.html
-        - title: Product 1
-          url: /p1_landing_page.html
-        - title: Product 2
-          url: /p2_landing_page.html
+#topnav_dropdowns:
+#- title: Topnav dropdowns
+#  folders:
+#    - title: VoxelShop Resources
+#      folderitems:
```
Comment (simlu) [inline _data/topnav.yml]: Again, why is this commented out?

**Comment by simlu** [comment]
@MrTopom Is this good to go or do you want to make further alterations before I put it live?

**Comment by MrTopom** [comment]
No I have to look for image issue before.

**Comment by MrTopom** [comment]
Image issue is solved but not all pages containts are done. You still want to put online with partial or need to add content before ?

**Comment by simlu** [comment]
@MrTopom I'm gonna put this online now and remove the wiki... (Well, remove all pages and put a link)

**Pull request merged**